mcux/README

```diff
@@ -17,6 +17,7 @@ Status:
    ----------------------------------------------------------------------------
    LPC54114	SDK 2.6.0 (2019-06-14)
    LPC55S69	SDK 2.6.3 (2019-09-26)
+   K32L2B   SDK 2.6.0 (2019-10-18)
```

Comment:
Replace the spaces with a tab to fix the alignment of "SDK"

Reply:
updated, thanks.